src/aktualizr_repo/director_repo.cc

```diff
@@ -35,7 +35,7 @@ void DirectorRepo::signTargets() {
   }
 
   Utils::writeFile(path_ / "repo/director/targets.json",
-                   Utils::jsonToCanonicalStr(signTuf(Uptane::Role::Targets(), targets_unsigned)));
+                   Utils::jsonToCanonicalStr(signTuf("targets", targets_unsigned)));
```

Comment:
I would prefer not to eliminate the specialized type usage. I've expanded the `Role` enum class to support delegations. In a minute I'll refactor my code to pull out a commit with just that change and I'll put that on a branch that you can then rebase on top of.

Reply:
Here's the branch with the single useful commit: https://github.com/advancedtelematic/aktualizr/tree/feat/delegation-roles. If you rebase on that, then you can keep the `Uptane::Role` objects instead of using strings.